src/routes/routerConfig.tsx

```diff
@@ -68,6 +68,10 @@ export const playgroundOnlyRouterConfig: RouteObject[] = [
         path: 'playground',
         lazy: Playground
       },
+      {
+        path: 'playground/share/:uuid?',
+        lazy: Playground
+      },
```

Comment:
I thought the new share link generation will be for Source Academy @ NUS only? Why is it instead only in the public deployment route (and not for SA @ NUS)? @chownces can you confirm? I may not have the latest information from the meetings.

Reply:
Agree. Let's shift this to a 'with-backend' only router.